File: vote_vite_admin/src/openapi/models/EvaluationContentDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { EvaluationCategory } from './EvaluationCategory';

export type EvaluationContentDto = {
    id?: string;
    creationTime?: string;
    creatorId?: string | null;
    lastModificationTime?: string | null;
    lastModifierId?: string | null;
    name?: string | null;
    description?: string | null;
    category?: EvaluationCategory;
    concurrencyStamp?: string | null;
};

File: abp/src/Wallee.Boc.Vote.Application.Contracts/EvaluationContents/EvaluationContentDto.cs
using System;
using Volo.Abp.Application.Dtos;
using Volo.Abp.Domain.Entities;
using Wallee.Boc.Vote.Appraisements;

namespace Wallee.Boc.Vote.EvaluationContents
{
    public class EvaluationContentDto : AuditedEntityDto<Guid>, IHasConcurrencyStamp
    {
        /// <summary>
        /// 履职评价名称(项目)
        /// </summary>
        public string Name { get; set; } = null!;
        /// <summary>
        /// 履职评价描述(评价标准)
        /// </summary>
        public string Description { get; set; } = null!;
        /// <summary>
        /// 评价内容分类
        /// </summary>
        public EvaluationCategory Category { get; set; }
        public string ConcurrencyStamp { get; set; } = null!;
    }
}
